enhancements/update/automatic-updates.md

```diff
@@ -0,0 +1,166 @@
+
+There are also potential future mitigations:
+
+* The cluster-version operator could be taught to limit automatic upgrades to those which appear in the [signed metadata][cluster-version-operator-release-metadata] of either the source or target release.
+    The drawback to this approach is that upgrades would be limited to those expected when the releases were created and signed.
+
+## Design Details
+
+### Test Plan
+
+In addition to unit tests, this feature will be tested in a fleet of canary clusters which will run with automatic updates enabled.
+Having the canaries successfully and automatically update in to and out of a candidate release will be part of release and update stabilization criteria.
+
+### Graduation Criteria
+
+The ClusterVersion object is already GA, so there would be nothing to graduate or space to cool a preview implementation.
+
+### Upgrade / Downgrade Strategy
+
+The YAML rendering of the old and updated type are compatible, with the only difference being downgrades to earlier releases (where the property was not available) will clear any stored value.
+Upon upgrading back to a version where the property is available, administrators would need to manually set it to restore automatic upgrade behavior.
+
+### Version Skew Strategy
+
+As a boolean property, this would be [`false` by default][go-zero-values] on upgrades to the updated ClusterVersion Custom Resource Definition.
+This preserves semantics for existing clusters, where manual updates are the only option.
+
+The installer or administrator could set `automaticUpdates` `true` at cluster-creation time if they wished to have it enabled out of the box on new clusters.
+
+## Implementation History
+
+* [API pull request][api-pull-request].
+
+## Drawbacks
+
+The drawbacks are increased exposure to [automatic-update risks](#risks-and-mitigations).
+
+## Alternatives
+
+A [schedule structure][schedule-proposal] like:
+
+```yaml
+automaticUpdates:
+  schedule:
+  - Mon..Fri *-2..10-* 10:00,14:00
+  - Tues..Thurs *-11..1-* 11:00
+```
+
+The schedule structure was designed to support maintenance windows, allowing for updates on certain days or during certain times.
+But we could enforce maintenance windows on a cluster independently of configuring automatic updates.
+For example, a `schedule` property directly on [the ClusterVersion `spec`][api-spec] would protect administrators from accidentally using the web console or `oc adm upgrade ...` to trigger an upgrade outside of the configured window.
+Instead, administrators would have to adjust the `schedule` configuration or set an override option to trigger out-of-window updates.
+Customization like this can also be addressed by intermediate [policy-engine][cincinnati-policy-engine], without involving the ClusterVersion configuration at all.
+Teaching the cluster-version operator about a local `schedule` filter is effectively like a local policy engine.
```

Comment:
There is a difference though:
 * if a schedule is enforced locally by CVO, the operator can grow logic for an escape hatch for manual forced-updates (or only apply to auto-updates at all)
 * if a schedule is enforced globally by policy-engine, it cannot be force-bypassed locally

Reply:
> if a schedule is enforced globally by policy-engine, it cannot be force-bypassed locally

Sure it can, you just set the update by pullspec and force the "don't worry if this is not in your graph" exception.